compiler/crates/relay-config/src/typegen_config.rs

```diff
@@ -60,6 +60,25 @@ pub struct CustomTypeImport {
     pub path: PathBuf,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+#[serde(untagged)]
+pub enum ResolverContextTypeInput {
+    Path(ResolverContextTypeInputPath),
+    Package(ResolverContextTypeInputPackage),
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+pub struct ResolverContextTypeInputPath {
+    pub name: StringKey,
+    pub path: PathBuf,
+}
```

Comment:
```suggestion
/// Specifies how Relay can import the Resolver context type from a path
#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
pub struct ResolverContextTypeInputPath {
    /// The name under which the type is exported from the module
    pub name: StringKey,
    /// The path to the module
    pub path: PathBuf,
}
```

Can we also clarify what the path is relative to?